reana-ui/src/pages/workflowDetails/components/LauncherLabel.js

```diff
@@ -0,0 +1,46 @@
+/*
+  This file is part of REANA.
+  Copyright (C) 2022 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import PropTypes from "prop-types";
+import { Label } from "semantic-ui-react";
+
+export default function LauncherLabel({ url }) {
+  /**
+   * Return appropriate Label component props according to the given launcher URL.
+   * @param {String} url Submission workflow launcher URL.
+   * @returns {Object} Appropriate component props.
+   */
+  function getProps(url) {
+    const externalSrcProps = {
+      as: "a",
+      href: url,
+      target: "_blank",
+      rel: "noopener noreferrer",
+    };
+
+    if (!url) {
+      return { content: "reana-client", icon: "terminal" };
```

Comment:
BTW I'm not sure about the terminal label. Even if the workflow was started from Zenodo, or from GitHub, people can always use `reana-client` to work with it. They can inspect status and progress, download files, etc.  Moreover, the terminal label is not "clickable", so it may be confusing to just display it without anything.

I would suggest to keep the display as it was in the past for those workflows where `launcher_url` is not set, and to display label only for those where the origin was non-reana-client and where we can link back to the source.

![b1](https://user-images.githubusercontent.com/517546/157293403-7dedfa6e-fcca-4382-990b-a90de67b5a08.png)